.github/workflows/fuzz-test.yml

```diff
@@ -0,0 +1,32 @@
+---
+name: Fuzz tests
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - main
+      - develop  
+  workflow_dispatch:
+  workflow_call:
+
+
+jobs:
+  fuzz_test:
+    name: Polygon Edge
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.18.x
+
+      - name: Checkout Code
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
+      - name: Install Dependencies
+        run: ./setup-ci.sh
+
+      - name: Run Fuzz Test
```

Comment:
nitpick
```suggestion
      - name: Run Fuzz Tests
```

Reply:
Name for UTs is not plural, so to follow the convetion